exercise_3/ShowCommand.hpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by denys on 18.07.19.
+//
+
+#pragma once
+
+
+#include "ICommand.hpp"
+#include "RecordsHandler.hpp"
+
+class ShowCommand : public ICommand{
+
+private:
+    std::istringstream &_command;
+    RecordsHandler &_recordsHandler;
+
+public:
+    static const std::string    COMMAND_NAME;
+
+    static bool can_execute(const std::string &command_name) {
```

Comment:
Such implementation isn't particularly user-friendly (it requires including given command's header, knowing its exact type, and implies some typing).